htdocs/app/Migrations/Version20190211224434.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20190211224434 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    : void {
+        $this->addSql('UPDATE `sys_menu` SET `href` = `articles.php?page=terms` WHERE id_string = `MNU_START_TOS`;');
```

Comment:
MNU_START_TOS is surrounded by backticks and must be surrounded by single quotes in order to pass travis-ci